rom/dev/tests/rom_integration_tests/test_update_reset.rs

```diff
@@ -388,3 +390,58 @@ fn test_update_reset_vendor_lms_pub_key_idx_dv_mismatch() {
         CaliptraError::IMAGE_VERIFIER_ERR_UPDATE_RESET_VENDOR_LMS_PUB_KEY_IDX_MISMATCH.into()
     );
 }
+
+#[test]
+fn test_check_rom_update_reset_status_reg() {
```

Comment:
Once the ROM Test document update PR is completed, can you add this test case to it and remove the scenario from the test gaps section?

Reply:
Should I wait for that PR to go in before merging this?

Reply:
I updated test-coverage.md accordingly